StyleCop.Analyzers/StyleCop.Analyzers/Helpers/TriviaHelper.cs

```diff
@@ -392,6 +392,93 @@ internal static bool HasBuiltinEndLine(this SyntaxTrivia trivia)
                 || trivia.IsKind(SyntaxKind.EndOfLineTrivia);
         }
 
+        /// <summary>
+        /// Checks whether the given trivia list contains one or more blank lines.
+        /// </summary>
+        /// <param name="triviaList">The trivia list to check.</param>
+        /// <param name="startsOnNewLine">Indicates if the trivia list starts on a new line.</param>
+        /// <returns>True if the given trivia list contains one or more blank lines.</returns>
+        internal static bool ContainsBlankLines(this IReadOnlyList<SyntaxTrivia> triviaList, bool startsOnNewLine = true)
```

Comment:
:question: This method ignores blank lines in some cases. Should we make a note of this in the method documentation?

* Excluded regions (see `GetExcludedSpans` in #2034)
* Documentation comments (at ones using `/** */`, and maybe ones using `///`)
* Block comments (`/* ... */`)